### PR TITLE
New time card format

### LINKED ISSUE #24: Find a new format to use for the Time Cards
Find a new format to use for the Time Cards to better parse them, create them, and edit them.

### PR DESCRIPTION
**Pull request by NLSteveO**
Updated the file format of my input files to use TOML and updated the unit tests and README to reflect the change.

Closes #24

### CODE REVIEW

File: PunchCard.py [hunk 1]
```diff
@@ -56,14 +57,16 @@ def printWeekHours(hours, timeFormat):
...
+def main(config, timeFormat):  # pragma: no cover
+    daysOfTheWeek = ['saturday', 'sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday']
     weekHours = 0.0
-    print(input.pop(0))
-    for line in input:
-        daySplit = line.split(',')
-        dayLetter = daySplit.pop(0)[0]
-        dayHours = calculateDay(daySplit)
-        printDaysHours(dayLetter, dayHours, timeFormat)
+    print(config['title']+'\n')
```
Comment (whymarrh) [inline PunchCard.py]: Nit: spaces around `+`

File: PunchCard.py [hunk 2]
```diff
@@ -56,14 +57,16 @@ def printWeekHours(hours, timeFormat):
...
+    for day in daysOfTheWeek:
+        if day not in config['day'] or config['day'][day] == {}:
```
Comment (whymarrh) [inline PunchCard.py]: Nit: is `config['day'][day] == {}` supposed to check if `config['day'][day]` is a dict or if `config['day'][day]` is empty?

Comment (NLSteveO) [reply PunchCard.py]: If it is empty.

**Comment by NLSteveO** [comment]
@MitMaro

File: PunchCard.py (hunk 2 above)
Comment (whymarrh) [reply PunchCard.py]: I think `not config['day'][day]` might be a bit more idiomatic—I could be wrong though

**Review by MitMaro — COMMENTED** [review]
Looks good to me! Nothing else to add that @whymarrh didn't already mention.

**Pull request merged**